schema/annotation-source.yaml

```diff
@@ -56,6 +56,11 @@ $defs:
           - $ref: "#/$defs/Document"
           - $ref_curie: gks.core:CURIE
         description: A document in which the information content is expressed.
+      information_quality:
```

Comment:
Adding the core `information_quality` attribute to InformationEntity, which will be extended to `statement_quality` in the Statement class to hold clinvar review status info.

Reply:
@ahwagner and I discussed eventually the clinvar.review_status into an `evidence_level` field which we think is more reflective of the Statement's `source_level` or `source_category`.  Since some statements are Evidence and some are Assertions, and some are ???, we think that having `level` and `score` and `direction` replicated for the different uses in a single super class called Statement is a bit confusing. In some future world once we have broad adoption we can consider moving towards a more complex model whereby we can capture all the various aspects of all the various statements. For now, know the Statement.level or Statement.source_category would be fine. And, it may make sense to transform ClinVar's review status to this.

I will consider looking into using the `statement_quality` attribute but I will need to put in whatever clinvar states and not modify it. If that is allowed then I'm not so certain I see the big difference with extensions vs `statement_quality`

Reply:
@larrybabb Sorry if I missed something, but what is this business about a Statement source_level  or source_category attribute?   I don't recall seeing this in any of the schema or examples.  What is this attribute intended to capture exactly? 

Also not following what this topic has to do with level, score, and direction attributes (by which I assume you are referring to the confidence and evidence level/score/direction modeling in PR #88?)

Reply:
i presumed Alex had already commented on PR #88 re: moving away from two streams of attributes (one for evidence and one for confidence). I was referencing a decision that Alex and I discussed prior to writing this up. Sorry for the confusion we will clarify today when we meet.

Reply:
We are proposing that the `level` attribute is really about normalizing a category about the level of the `source` that supplies the evidence or makes the assertion.  Since an "assertion" statement would depend on the `category` or `level` of the `source` of the assertion similar to how an "data item/evidence" statement would depend on the `category` or `level` of the `source` of the data item, the `level` should be a singular attribute on Statement that relates to the `level` or `category` of the source of the information.  If we do this then we could normalize the ClinVar review status as the `source_level` or `source_category` similar to how the civic EID evidence `level` terms are being normalized. We would have to build a set of terms that would allow these `categories` of information sources to be comparable (which Alex is working on).

Reply:
> re: moving away from two streams of attributes (one for evidence and one for confidence).

So in our last talk we agreed it was important to explicitly distinguish between assessment of evidence strength/level, and assessment of confidence level - as these are different things, and there are some statements that assess confidence level (e.g. ClinVar SCVs) and others that assess evidence strength (e.g. CIViC TR, and many other ClinGen curation efforts such as Gene Validity, Dosage Sensitivity, etc).  There have been no comments on #88 indicating such a change in thinking - but should I take your comment to mean that now you guys are now wanting to collapse the distinction?